cmd/werf/common/common.go

```diff
@@ -99,6 +99,11 @@ func SetupConfigTemplatesDir(cmdData *CmdData, cmd *cobra.Command) {
 	cmd.Flags().StringVarP(cmdData.ConfigTemplatesDir, "config-templates-dir", "", os.Getenv("WERF_CONFIG_TEMPLATES_DIR"), `Custom configuration templates directory (default $WERF_CONFIG_TEMPLATES_DIR or .werf in working directory)`)
 }
 
+func SetupConfigRenderPath(cmdData *CmdData, cmd *cobra.Command) {
```

Comment:
Let's rename `SetupConfigRenderPath` to `SetupConfigRenderDir`  to keep consistent naming.